verification/contextual/src/contextual_block_verifier.rs

```diff
@@ -408,18 +416,25 @@ impl<'a, CS: ChainStore<'a>> BlockTxsVerifier<'a, CS> {
             })
             .collect::<Result<Vec<(Byte32, CacheEntry)>, Error>>()?;
 
-        let sum: Cycle = ret.iter().map(|(_, cache_entry)| cache_entry.cycles).sum();
-        let cache_entires = ret
+        let sum: Cycle = ret
             .iter()
+            .skip(1)
+            .map(|(_, cache_entry)| cache_entry.cycles)
+            .sum();
+        let cache_entires: Vec<CacheEntry> = ret
+            .iter()
+            .skip(1)
             .map(|(_, cache_entry)| cache_entry)
             .cloned()
             .collect();
-        handle.spawn(async move {
-            let mut guard = txs_verify_cache.write().await;
-            for (k, v) in ret {
-                guard.put(k, v);
-            }
-        });
+        if !cache_entires.is_empty() {
+            handle.spawn(async move {
+                let mut guard = txs_verify_cache.write().await;
+                for (k, v) in ret {
```

Comment:
Here, when there is a transaction, the cellbase is still stuffed in the cache, so should skip 1

Reply:
it has been skipped in line 426

Reply:
> it has been skipped in line 426

ret len = 2, cache_entires len is 1, for loop ret kv insert, not skip

Reply:
fixed, now changed to skip cellbase result in line 417, please review again, thanks.